README.md

```diff
@@ -1 +1,27 @@
-# fiatconnect-types
\ No newline at end of file
+# fiatconnect-types
```

Comment:
Maybe add a section here (primarily for our own benefit) about how to deploy/upload a new version of the package to NPM?